src/test/java/org/sasanlabs/service/vulnerability/commandInjection/CommandInjectionTest.java

```diff
@@ -0,0 +1,592 @@
+        ResponseEntity<GenericVulnerabilityResponseBean<String>> responseEntity =
+                commandInjection.getVulnerablePayloadLevel2(LOCALHOST, entity);
+
+        // Assert
+        assertStatusAndValidate(responseEntity, LOCALHOST, mockedSuccessPingResponse);
+
+        verify(commandInjection).getVulnerablePayloadLevel2(eq(LOCALHOST), eq(entity));
+    }
+
+    @Test
+    public void getVulnerablePayloadLevel2_LocalhostAnd3b_ExpectPingSuccess()
+            throws ServiceApplicationException, IOException {
+        RequestEntity entity =
+                new RequestEntity(
+                        HttpMethod.GET,
+                        URI.create(
+                                "http://localhost:9090/CommandInjection/LEVEL_2?ipaddress=localhost%3b"));
+        ResponseEntity<GenericVulnerabilityResponseBean<String>> responseEntity =
+                commandInjection.getVulnerablePayloadLevel2(LOCALHOST, entity);
+
+        // Assert
+        assertStatusAndValidate(responseEntity, LOCALHOST, mockedSuccessPingResponse);
+
+        verify(commandInjection).getVulnerablePayloadLevel2(eq(LOCALHOST), eq(entity));
+    }
+
+    @Test
+    public void getVulnerablePayloadLevel3_NoIp_ExpectNoResponseContent()
+            throws ServiceApplicationException, IOException {
+        RequestEntity entity = new RequestEntity(HttpMethod.GET, URI.create(Strings.EMPTY));
+        ResponseEntity<GenericVulnerabilityResponseBean<String>> responseEntity =
+                commandInjection.getVulnerablePayloadLevel3(Strings.EMPTY, entity);
+
+        // Assert
+        assertStatusAndValidate(responseEntity, Strings.EMPTY, Strings.EMPTY);
+
+        verify(commandInjection).getVulnerablePayloadLevel3(eq(Strings.EMPTY), eq(entity));
+    }
+
+    @Test
+    public void getVulnerablePayloadLevel3_LocalhostAndSEMICOLON_ExpectNoResponseContent()
+            throws ServiceApplicationException, IOException {
+        RequestEntity entity =
+                new RequestEntity(
+                        HttpMethod.GET,
+                        URI.create(
+                                "http://localhost:9090/CommandInjection/LEVEL_3?ipaddress=localhost;"));
+        ResponseEntity<GenericVulnerabilityResponseBean<String>> responseEntity =
+                commandInjection.getVulnerablePayloadLevel3(LOCALHOST, entity);
+
+        // Assert
+        assertStatusAndValidate(responseEntity, LOCALHOST, Strings.EMPTY);
+
+        verify(commandInjection).getVulnerablePayloadLevel3(eq(LOCALHOST), eq(entity));
+    }
+
+    @Test
+    public void getVulnerablePayloadLevel3_LocalhostPlusAnd_ExpectNoResponseContent()
+            throws ServiceApplicationException, IOException {
+        RequestEntity entity =
+                new RequestEntity(
+                        HttpMethod.GET,
+                        URI.create(
+                                "http://localhost:9090/CommandInjection/LEVEL_3?ipaddress=localhost&"));
```

Comment:
you can write a method which returns various response entities, call that method get the response entity as same line is repeated multiple times.